docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss

```diff
@@ -0,0 +1,159 @@
+$hb-sidebar-width: 16.5rem;
```

Comment:
sorry for renaming this.. make sure to compare to the original file `three_column_with_image.scss`

Reply:
Renaming is fine but we should move variable declarations to `/settings/variables.general.scss`.